fbw-common/src/systems/navdata/shared/types/Airport.ts

```diff
@@ -45,6 +45,11 @@ export interface Airport extends DatabaseItem<SectionCode.Airport> {
    */
   distance?: NauticalMiles;
 
+  /**
+   * Magnetic variation at the airport
```

Comment:
If we keep this type, make sure to note that it's MSFS2024 only.

Reply:
I got rid of the field now, since it's MSFS2024 only and really just a liability for the scenery issues you mention.